packages/@tailwindcss-vite/src/index.ts

```diff
@@ -193,7 +192,9 @@ export default function tailwindcss(): Plugin[] {
           // The reason why we can not rely on the invalidation here is that the
           // users would otherwise see a flicker in the styles as the CSS might
           // be loaded with an invalid set of candidates first.
-          await server?.waitForRequestsIdle?.(id)
+          for (let server of servers) {
+            await server.waitForRequestsIdle(id)
+          }
```

Comment:
Should we do a `Promise.all` here?

Reply:
Yeah probably better to start waiting immediately 👍